doc/api/stream.md

```diff
@@ -2447,37 +2447,6 @@ import { Readable } from 'node:stream';
 await Readable.from([1, 2, 3, 4]).take(2).toArray(); // [1, 2]
 ```
 
-##### `readable.asIndexedPairs([options])`
-
-<!-- YAML
-added:
-  - v17.5.0
-  - v16.15.0
-changes:
-  - version: REPLACEME
```

Comment:
nit: It'd be a good idea to wait for this change to be in a release to minimize the conflicts.

Reply:
I added blocked preemptively

Reply:
@adhu95 are we good to go with this PR? 🙂

Reply:
I think we are, it'd just need a rebase to solve the git conflict :)

Reply:
Oh I looked from the phone and missed the conflicts 😯
Will handle this, thanks 🙂